project/Dependencies.scala

```diff
@@ -280,20 +280,20 @@ object Dependencies {
     libraryDependencies ++= Seq(
         "com.typesafe.akka" %% "akka-http" % AkkaHttpVersion, // ApacheV2
         "com.github.matsluni" %% "aws-spi-akka-http" % AwsSpiAkkaHttpVersion excludeAll // ApacheV2
-        (
-          ExclusionRule(organization = "com.typesafe.akka")
-        ),
-        "software.amazon.awssdk" % "kinesis" % AwsSdk2Version excludeAll // ApacheV2
-        (
-          ExclusionRule("software.amazon.awssdk", "apache-client"),
-          ExclusionRule("software.amazon.awssdk", "netty-nio-client")
-        ),
-        "software.amazon.awssdk" % "firehose" % AwsSdk2Version excludeAll // ApacheV2
+        ExclusionRule(organization = "com.typesafe.akka"),
+        "org.mockito" % "mockito-core" % mockitoVersion % Test // MIT
+      ) ++ Seq(
+        "software.amazon.awssdk" % "kinesis" % AwsSdk2Version, // ApacheV2
+        "software.amazon.awssdk" % "firehose" % AwsSdk2Version, // ApacheV2
+        "software.amazon.awssdk" % "dynamodb" % AwsSdk2Version, // ApacheV2
```

Comment:
DynamoDB is only used in tests, right?

Reply:
Is used by the KCL library, it's a transitive dependency. Probably I should remove it.

Reply:
Ah, OK.
Might be better to pull it transitively, so please remove it.